src/CommitmentsV2/SFA.DAS.CommitmentsV2.MessageHandlers.TestHarness/Program.cs

```diff
@@ -42,7 +42,9 @@ public static async Task Main()
                 .UseMessageConventions()
                 .UseNewtonsoftJsonSerializer();
 
-            endpointConfiguration.Conventions().DefiningEventsAs(t => t == typeof(RecordedAct1CompletionPayment));
+            endpointConfiguration.Conventions().DefiningEventsAs(t =>
```

Comment:
Why this is here now?

Reply:
Let me re-word in English: why is this here now?

Reply:
I wasn't able to publish the event from test harness without making this change.